pkgs/by-name/me/meson/setup-hook.sh

```diff
@@ -52,16 +64,18 @@ mesonCheckPhase() {
 mesonInstallPhase() {
     runHook preInstall
 
-    # shellcheck disable=SC2086
-    local flagsArray=($mesonInstallFlags "${mesonInstallFlagsArray[@]}")
+    local flagsArray=()
+
+    flagsArray+=($mesonInstallFlags "${mesonInstallFlagsArray[@]}")
```

Comment:
don't think this makes a difference but whatever

Reply:
Future-proof.

The `$meson...` vars should be added last, and when in the future they change it can be done easier.